docs/user.md

```diff
@@ -6,9 +6,9 @@ slug: "/"
 Welcome to the DAppNode user guide.
```

Comment:
```suggestion
Welcome to the Dappnode User Guide!
```

Reply:
That's already resolved. Make sure your commit filter is set to newest.

Reply:
Ughh. Idk how that happened. I clicked to open what was the most recent requested review when I was working on it last night. I Thought it was redundant and had been done but just kept going.